test/typing/transformers.test-d.ts

```diff
@@ -0,0 +1,31 @@
+import { expectType } from "tsd";
+import Redis from "../../built";
+
+interface User {
+  name: string;
+  title: string;
+}
+
+const user: User = { name: "Bob", title: "Engineer" };
+const map = new Map([["key", "value"]]);
```

Comment:
I think a test with `new Map([[1, "value"]]);` and `new Map([[Buffer.from('key'), "value"]]);` would be good

Reply:
Added to both typing tests and functional tests.